source/people/aaswenson.rst

```diff
@@ -0,0 +1,29 @@
+Alexander Swenson
+=============
+
+Contact Information
+-------------------
+| `E-mail <aaswenson@wisc.edu>`_
+
+----
```

Comment:
@aaswenson - see my comment. Blank line is necessary for division. The dashed like that is currently there is not necessary.